nautobot/extras/api/views.py

```diff
@@ -764,7 +815,11 @@ class JobRunTokenPermissions(TokenPermissions):
 
     @extend_schema(
         methods=["post"],
-        request=serializers.JobInputSerializer,
+        request={
+            "application/json": serializers.JobInputSerializer,
+            "application/x-www-form-urlencoded": serializers.JobInputSerializer,
```

Comment:
I'm curious about this one - does it actually work?

Reply:
Yup! Gotta update the PR with screenshots

Reply:
Proposed change:

```diff
diff --git a/nautobot/core/settings.py b/nautobot/core/settings.py
index 235c4fccc..d20e18743 100644
--- a/nautobot/core/settings.py
+++ b/nautobot/core/settings.py
@@ -189,6 +189,7 @@ REST_FRAMEWORK = {
         "rest_framework.renderers.JSONRenderer",
         "nautobot.core.api.renderers.FormlessBrowsableAPIRenderer",
     ),
+    "DEFAULT_PARSER_CLASSES": ("rest_framework.parsers.JSONParser",),
     "DEFAULT_SCHEMA_CLASS": "nautobot.core.api.schema.NautobotAutoSchema",
     # Version to use if the client doesn't request otherwise.
     # This should only change (if at all) with Nautobot major (breaking) releases.
diff --git a/nautobot/extras/api/views.py b/nautobot/extras/api/views.py
index f1817ba6b..5b7062772 100644
--- a/nautobot/extras/api/views.py
+++ b/nautobot/extras/api/views.py
@@ -12,6 +12,7 @@ from graphql import GraphQLError
 from rest_framework import status
 from rest_framework.decorators import action
 from rest_framework.exceptions import MethodNotAllowed, PermissionDenied, ValidationError
+from rest_framework.parsers import JSONParser, MultiPartParser
 from rest_framework.response import Response
 from rest_framework.routers import APIRootView
 from rest_framework import mixins, viewsets
@@ -817,12 +818,16 @@ class JobViewSet(
         methods=["post"],
         request={
             "application/json": serializers.JobInputSerializer,
-            "application/x-www-form-urlencoded": serializers.JobInputSerializer,
             "multipart/form-data": serializers.JobMultiPartInputSerializer,
         },
         responses={"201": serializers.JobRunResponseSerializer},
     )
-    @action(detail=True, methods=["post"], permission_classes=[JobRunTokenPermissions])
+    @action(
+        detail=True,
+        methods=["post"],
+        permission_classes=[JobRunTokenPermissions],
+        parser_classes=[JSONParser, MultiPartParser],
+    )
     def run(self, request, *args, pk, **kwargs):
         """Run the specified Job."""
         job_model = self.get_object()
@@ -841,6 +846,7 @@ class JobViewSet(
         permission_classes=[JobRunTokenPermissions],
         url_path="(?P<class_path>[^/]+/[^/]+/[^/]+)/run",  # /api/extras/jobs/<class_path>/run/
         url_name="run",
+        parser_classes=[JSONParser, MultiPartParser],
     )
     def run_deprecated(self, request, class_path):
         """
```

Reply:
I see you `MultiPartParser`!

Reply:
@jathanism @glennmatthews this is done.